packages/data-validation/package.json

```diff
@@ -0,0 +1,76 @@
+{
+    "name": "@univerjs/data-validation",
+    "version": "0.1.0-beta.4",
```

Comment:
`"version"` need to be updated to the version in which you would like to release your feature.